index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+
+   <h1>Dr. Norman Borlaug</h1>
+   <div id="img-div">
+
+    <img src="dr.png" alt="" id="img">
```

Comment:
- [ ] it's best practice to add a descriptive `alt=""` inside the `img` tag. I kindly suggest you to add a description to your img. for example: `<img scrc="dr.png" alt="this is a doctor picture" id="img">`